front/app/containers/App/VerificationModal/index.tsx

```diff
@@ -2,7 +2,8 @@ import React, { useCallback, useState, useEffect } from 'react';
 
 // components
 import Modal from 'components/UI/Modal';
-import VerificationSteps from 'components/SignUpIn/SignUp/VerificationSignUpStep/VerificationSteps';
+// TODO: Change when we move to container
```

Comment:
Change what?

Reply:
Change the import because they will live in the same container.